rpms/generate_rpm_package.sh

```diff
@@ -61,12 +61,12 @@ build_rpm() {
         VERSION="$(cat ${SOURCES_DIRECTORY}/src/VERSION | cut -d 'v' -f 2)"
         echo "Version is $VERSION"
         if [[ "${TARGET}" == "manager" ]] && [[ "${LEGACY}" == "yes" ]]; then
-            MAJOR_MINOR="$(echo ${VERSION} | cut -d "v" -f 2)"
+            MAJOR_MINOR="$(echo ${VERSION}|cut -d . -f 1-2)"
             echo "major minor is $MAJOR_MINOR"
-            if [[ "${MAJOR_MINOR}" > "3.9" ]] || [[ "${MAJOR_MINOR}" == "3.9" ]]; then
+            if [[ $(echo "$MAJOR_MINOR 3.9" | awk '{print ($1 > $2)}') ]] || [[ "${MAJOR_MINOR}" == "3.9" ]]; then
                 echo "Wazuh Manager is not supported for CentOS 5 from v3.9.0."
                 echo "Version to build: ${VERSION}."
-                exit 1
+                return 1
```

Comment:
This return should be a `clean 1`.

Reply:
In this commit https://github.com/wazuh/wazuh-packages/pull/322/commits/7ce7042530bc906ff9d6dc27c4296d77e592ad22, I changed the `exit 1` and `clean 1` calls in build and build_rpm functions for `return 1`  this way when something fails, these functions will return 1 and this line will call to clean:
https://github.com/wazuh/wazuh-packages/blob/7ce7042530bc906ff9d6dc27c4296d77e592ad22/rpms/generate_rpm_package.sh#L264